### PR TITLE
Minor updates to PWA IA & code cleanup

### PR DESCRIPTION
**Pull request by petele**
Changes proposed in this pull request:
- Minor updates to the PWA landing page TOC
- Update installability code snippets to better reflect best practices
- Update note about changes to installability criteria with link to updated docs
- Update note about manifest changes and how they affect installed PWAs

### CODE REVIEW

File: src/site/_data/paths/progressive-web-apps.json [hunk 1]
```diff
@@ -40,18 +40,26 @@
       ]
     },
     {
-      "title": "Capable",
+      "title": "Create an app like user experience",
```
Comment (jpmedley) [inline src/site/_data/paths/progressive-web-apps.json:43]:
```suggestion
      "title": "Create an app-like user experience",
```

File: src/site/content/en/progressive-web-apps/customize-install/index.md [hunk 2]
```diff
@@ -69,6 +69,7 @@ event, and update your user interface to indicate that the user can install
 your PWA. This is highlighted below.
 
 ```js
+// Create deferredPrompt for use later to show browser install prompt.
```
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/customize-install/index.md:72]: I don't follow.

File: src/site/content/en/progressive-web-apps/customize-install/index.md [hunk 3]
```diff
@@ -133,9 +134,13 @@ whenever your PWA is installed, no matter what mechanism is used to install
 your PWA.
 
 ```js
-window.addEventListener('appinstalled', (evt) => {
-  // Log install to analytics
-  console.log('INSTALL: Success');
+window.addEventListener('appinstalled', () => {
+  // Hide the app provided install promotion
```
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/customize-install/index.md:138]:
```suggestion
  // Hide the app-provided install promotion
```

File: src/site/content/en/progressive-web-apps/customize-install/index.md [hunk 4]
```diff
@@ -133,9 +134,13 @@ whenever your PWA is installed, no matter what mechanism is used to install
...
+  hideInstallPromotion();
+  // Clear the deferredPrompt so it can be gc'ed
```
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/customize-install/index.md:140]: gc'ed?

File: src/site/content/en/progressive-web-apps/customize-install/index.md [hunk 5]
```diff
@@ -201,16 +202,6 @@ PWA, use conditional CSS:
...
-
-### Chrome on Desktop
-
-On Desktop, the manifest is not automatically updated, but this is planned for
-a future update.
-
-[update-flow]: https://developers.google.com/web/fundamentals/integration/webapks#update-webapk
+What if you need to update your app name, or provide new icons?
+Check out [How Chrome handles updates to the web app manifest](/manifest-updates/)
+to see when and how are those changes reflected in Chrome.
```
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/customize-install/index.md:207]:
```suggestion
to see when and how are those changes are reflected in Chrome.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/progressive-web-apps/customize-install/index.md (hunk 4 above)
Comment (petele) [reply src/site/content/en/progressive-web-apps/customize-install/index.md:140]: garbage collected

File: src/site/content/en/progressive-web-apps/customize-install/index.md (hunk 2 above)
Comment (petele) [reply src/site/content/en/progressive-web-apps/customize-install/index.md]: the explanation is in the paragraph before, I just added a comment here to make it more clear. also tweaked it to 'initialize'

File: src/site/content/en/progressive-web-apps/customize-install/index.md (hunk 4 above)
Comment (petele) [reply src/site/content/en/progressive-web-apps/customize-install/index.md]: changed to garbage collected

**Review by jpmedley — APPROVED** [review]

**Pull request merged**